examples/algorithm_progression.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import concurrent.futures
+import inspect
+import itertools
+import os
+from pathlib import Path
+
+import algorithms
+import numpy as np
+import yaml
+
+import large_image
+
+VARIABLE_LAYERS = ['z', 'c', 't']
+
+
+def create_argparser():
+    argparser = argparse.ArgumentParser(
+        prog='Algorithm Progression',
+        description='Apply an algorithm to an input image '
+                    'for every parameter set in a given parameter space',
+    )
+    argparser.add_argument(
+        'algorithm_code',
+        choices=algorithms.ALGORITHM_CODES.keys(),
+        help='Code to specify which of the available algorithms should be used',
+    )
+    argparser.add_argument('input_filename', help='Path to an image to use as input')
+    argparser.add_argument(
+        '-o',
+        '--output_dir',
+        required=False,
+        help='Name for a new directory in this location, wherein result images will be stored',
+    )
+    argparser.add_argument(
+        '-w',
+        '--num_workers',
+        required=False,
+        default=4,
+        type=int,
+        help='Number of workers to use for processing alogrithm iterations',
```

Comment:
s/alogrithm/algorithm/

Reply:
Changed in [9d45193](https://github.com/girder/large_image/pull/1051/commits/9d45193e21b46b6121053daf87825898fdbcce1f)